dataeng/jobs/analytics/TerminateCluster.groovy

```diff
@@ -16,7 +16,7 @@ class TerminateCluster {
             steps {
                 virtualenv {
                     nature("shell")
-                    command('EXTRA_VARS="name=$CLUSTER_NAME" make terminate.emr')
+                    command('EXTRA_VARS="name=$CLUSTER_NAME" cd edx-analytics-configuration && make terminate.emr')
```

Comment:
Can we use $WORKSPACE here ?

Reply:
Also, the repo gets checked out in analytics-configuration folder.

Reply:
good catch